CoC.txt

```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
 they feel someone has violated this Code of Conduct, or if they have questions
 or concerns.
 
+================================================================================
+Behavioral Guidelines
+================================================================================
+
+As a member of the community, we ask you to:
```

Comment:
I would skip this because the CoC could apply in contexts where _community_ has to be defined and the meaning of _we_ is also subject to interpretation. For instance when this CoC is used by FPF as a non-profit, the **we** is different from when this CoC is used by groups of people who have no ties with FPF.